app/assets/javascripts/hidden_until_select.js.coffee

```diff
@@ -0,0 +1,16 @@
+class HiddenUntilSelect
+  constructor: ($root, $hidden_el) ->
+    @$hidden_el = $hidden_el
```

Comment:
Also, `@$hidden_el` is a somewhat misleading name for an element that will eventually be shown. Maybe `@$dependent`?